src/stan/services/arguments/arg_softabs_alpha.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef STAN_SERVICES_ARGUMENTS_ARG_SOFTABS_ALPHA_HPP
+#define STAN_SERVICES_ARGUMENTS_ARG_SOFTABS_ALPHA_HPP
+
+#include <stan/services/arguments/singleton_argument.hpp>
+
+namespace stan {
+  namespace services {
+
+    class arg_softabs_alpha: public real_argument {
+    public:
+      arg_softabs_alpha(): real_argument() {
+        _name = "alpha";
```

Comment:
(fix now)

1. no initial underscores

2.  these should be initializers, not assignments in the body